### PR TITLE
Added routing + entities + controller for dataset endpoint

### PR DESCRIPTION
**Pull request by ds161833**
This pr does not complete the feature #53, but just provides some little basic structure other devs can work with

### CODE REVIEW

File: Server/src/database.ts [hunk 1]
```diff
@@ -1,41 +1,41 @@
 import {createConnection} from 'typeorm';
```
Comment (Mehrdadmaskull) [inline Server/src/database.ts:1]: I say we keep this file as is. The changes really don't need to be in the commit.

Comment (ds161833) [reply Server/src/database.ts:1]: i mean i just commented it out, the code is still there + in my DatasetEntity I implemented basic typeorm sql operations

**Review by GD-Dheer — APPROVED** [review]
This looks good. Feature wasnt tested but we can build on this for an SOA.

wtf right haha

**Comment by JeremiahTee** [comment]
Went over the changes with @ds161833 over a call. He explained his code thoroughly, and went over some details that need to be refined. Approved PR

**Review by JeremiahTee — APPROVED** [review]

**Pull request merged**